phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:201:27: error: use of undeclared identifier 'eos_sc'

Just did recursive clone on 2386c863aae837288c3a354d067088a788a26fdf and compiled on Apple M1 with `cmake ..  -DPHOEBUS_ENABLE_HDF5=OFF`.

This is the error:
```
[ 66%] Building CXX object src/CMakeFiles/phoebus.dir/microphysics/eos_phoebus/eos_phoebus.cpp.o
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:172:19: error: use of undeclared identifier 'StellarCollapse'
  if (eos_type == StellarCollapse::EosType()) {
                  ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:185:33: error: expected ';' after expression
    singularity::StellarCollapse eos_sc(pin->GetString(block_name, "filename"),
                                ^
                                ;
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:185:18: error: no member named 'StellarCollapse' in namespace 'singularity'
    singularity::StellarCollapse eos_sc(pin->GetString(block_name, "filename"),
    ~~~~~~~~~~~~~^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:185:34: error: use of undeclared identifier 'eos_sc'
    singularity::StellarCollapse eos_sc(pin->GetString(block_name, "filename"),
                                 ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:196:27: error: use of undeclared identifier 'eos_sc'
    params.Add("sie_min", eos_sc.sieMin()/sie_unit);
                          ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:197:27: error: use of undeclared identifier 'eos_sc'
    params.Add("sie_max", eos_sc.sieMax()/sie_unit);
                          ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:198:25: error: use of undeclared identifier 'eos_sc'
    params.Add("T_min", eos_sc.TMin()/T_unit);
                        ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:199:25: error: use of undeclared identifier 'eos_sc'
    params.Add("T_max", eos_sc.TMax()/T_unit);
                        ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:200:27: error: use of undeclared identifier 'eos_sc'
    params.Add("rho_min", eos_sc.rhoMin()/rho_unit);
                          ^
/Users/jehammond/Work/DOE/phoebus/src/microphysics/eos_phoebus/eos_phoebus.cpp:201:27: error: use of undeclared identifier 'eos_sc'
    params.Add("rho_max", eos_sc.rhoMax()/rho_unit);
                          ^
10 errors generated.
```

--- a/src/microphysics/eos_phoebus/eos_phoebus.cpp
+++ b/src/microphysics/eos_phoebus/eos_phoebus.cpp
@@ -202,7 +202,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
     
     sie_min = eos_sc.sieMin()/sie_unit;
     sie_max = eos_sc.sieMax()/sie_unit;
-    T_min = seos_sc.TMin()/T_unit;
+    T_min = eos_sc.TMin()/T_unit;
     T_max = eos_sc.TMax()/T_unit;
     rho_min = eos_sc.rhoMin()/rho_unit;
     rho_max = eos_sc.rhoMax()/rho_unit;

